web/webplatform/fetch.sh

```diff
@@ -4,13 +4,18 @@ set -euo pipefail
 
 cd "$(dirname "$0")"
 
-VERSION="$(cut -d\  -f 1 < VERSION)"
-EXPECTED_SHA="$(cut -d\  -f 2 < VERSION)"
+VERSION="$(awk '{ print $1 }' < VERSION)"
+EXPECTED_SHA="$(awk '{ print $2 }' < VERSION)"
 FILENAME="autokitteh-web-v${VERSION}.zip"
 
 echo "want version ${VERSION}."
 
 checksum() {
+    if [[ -z "${EXPECTED_SHA}" ]]; then
+        echo "no checksum provided"
+        return
+    fi
+
     echo "verifying checksum..."
 
     sha=$(shasum -a 256 "$1" | cut -d ' ' -f 1)
```

Comment:
This works only because it's the first field. Use awk here as well?

Reply:
done